charter.html

```diff
@@ -299,6 +299,10 @@ <h2>Success Criteria</h2>
       Recommendation Snapshot</a>, and <a href="https://www.w3.org/2021/Process-20211102/#candidate-recommendation-draft">Candidate
       Recommendation Draft</a>. The WG does not intend to publish specifications as <a href="https://www.w3.org/2021/Process-20211102/#RecsPR">Proposed Recommendations</a>.
 
+    <p>
+      Each specification should to demonstrate being agnostic and independent to underlying systems and third-parties, such as aggregation or multi-party computation (MPC) services.
```

Comment:
This is a little broken, so I'd probably go with:

```suggestion
      Each specification should be independent of the technology that might be used to implement it, such as aggregation using trusted execution environments (TEE) or multi-party computation (MPC) services.
```

That said, I don't think that this is a necessary statement to make.  In fact, I would prefer that this NOT be in the charter.  It's something that a working group can -- and should -- discuss, rather than have it stipulated in a charter.